ui/tests/acceptance/enterprise-control-groups-test.js

```diff
@@ -13,7 +13,7 @@ import consoleClass from 'vault/tests/pages/components/console/ui-panel';
 import authForm from 'vault/tests/pages/components/auth-form';
 import controlGroup from 'vault/tests/pages/components/control-group';
 import controlGroupSuccess from 'vault/tests/pages/components/control-group-success';
-import { writeSecret } from 'vault/tests/helpers/kv/kv-run-commands';
+import { writeSecret } from 'vault/tests/helpers/kv/kv-selectors';
```

Comment:
I thought I remembered us talking about naming files `<resource>-helpers` even if it's redundant since they contain helper functions and selectors?